docs/development-guide/e2e-test-details.md

```diff
@@ -227,37 +227,58 @@ What we test: creating a `Paas` with the `ArgoCD` capability enabled.
 
 Scenarios:
 
-1. A minimal `Paas` with ArgoCD capability enabled.<br/><br/>
-   **Given** a minimal `Paas` and `ArgoCD` capability configuration,<br/>
-   **when** the minimal `Paas` is created with the `ArgoCD` capability enabled,<br/>
-   **then** the list entry in the applicationset should have been created,<br/>
-   **and** a namespace with the name `paasname-argocd` should have been created,<br/>
-   **and** an ArgoCD Application should have been created in namespaces,<br/>
-   **and** a quota with the name `paasname-argocd` should have been created,<br/>
-   **and** the ArgoCD Application and quota conform to the points below.
+1.  A minimal `Paas` with ArgoCD capability enabled.<br/><br/>
+    **Given** a minimal `Paas` and `ArgoCD` capability configuration,<br/>
+    **when** the minimal `Paas` is created with the `ArgoCD` capability enabled,<br/>
+    **then** the list entry in the applicationset should have been created,<br/>
+    **and** a namespace with the name `paasname-argocd` should have been created,<br/>
+    **and** an ArgoCD Application should have been created in namespaces,<br/>
+    **and** a quota with the name `paasname-argocd` should have been created,<br/>
+    **and** the ArgoCD Application and quota conform to the points below.
 
-   ArgoCD Application points:
+    ArgoCD Application points:
 
-   1. Assess gitUrl, path etc. exist in spec;
-   2. Assess RBAC .. determine how;
-   3. Assess Secrets exist in namespace and in argo...?
-   4. Assess Exclude appset is included in spec as ignoreDiff;
+        Quota points:
```

Comment:
This is now duplicated in lines 247 - 252 right?

Reply:
Yes.
Seems something I included twice.
Maybe an unexpected by effectt of a rebase, or something.